stable/redis/values.yaml

```diff
@@ -105,6 +105,7 @@ sentinel:
   service:
     ##  Redis Sentinel Service type
     type: ClusterIP
+    ClusterIP:
```

Comment:
can you also add this same changes from the values.yaml to the values-production.yaml?